entries/royalholloway.ac.uk.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Royal Holloway, University of London": {
+    "domain": "www.royalholloway.ac.uk",
+    "url": "https://www.royalholloway.ac.uk",
+    "tfa": [
+      "sms",
+      "call",
+      "totp"
+    ],
+    "documentation": "https://intranet.royalholloway.ac.uk/staff/it-services/multi-factor-authentication.aspx",
+    "keywords": [
+      "universities"
+    ],
+    "regions": [
+      "uk"
```

Comment:
**uk** isn't a real ISO-3166 code. **gb** is.
```suggestion
      "gb"
```

Reply:
Pretty odd but I see indeed they chose a strange abbr, whatever works.